### PR TITLE
Add search functionality

### LINKED ISSUE #182: Search for file/folder

### LINKED ISSUE #203: Make search field more responsive

### PR DESCRIPTION
**Pull request by tjomson**
- [x] Resolves #182 
- [x] Resolves #203

### CODE REVIEW

File: app/src/components/SearchBar.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+input {
+  border: 0;
+  width: 100%;
+}
+
+input:focus {
```
Comment (joglr) [inline app/src/components/SearchBar.css:6]: Hiding the outline of a focused search field is not good for accessibility
https://www.a11yproject.com/posts/never-remove-css-outlines/

File: app/src/components/SearchBar.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+input {
```
Comment (joglr) [inline app/src/components/SearchBar.css:1]: Could we use styled-components to keep the styling solution consistent? See examples in util.tsx

**Review by joglr — CHANGES_REQUESTED** [review]

**Comment by joglr** [comment]
Great addition! Will test it in a bit.

File: app/src/components/BubbleChart.css [hunk 3]
```diff
@@ -4,7 +4,7 @@
 }
 
 .file {
-  stroke-width: 0;
+  /* stroke-width: 0; */
```
Comment (joglr) [inline app/src/components/BubbleChart.css:7]: Commented out code

**Review by joglr — APPROVED** [review]

**Review by hojelse — CHANGES_REQUESTED** [review]
search_state

"Searching..." status is active just after reload

**Review by hojelse — APPROVED** [review]

**Pull request merged**